src/osd/ECUtil.h

```diff
@@ -32,6 +32,8 @@ const uint64_t CHUNK_INFO = 8;
 const uint64_t CHUNK_PADDING = 8;
 const uint64_t CHUNK_OVERHEAD = 16; // INFO + PADDING
 
+const std::string HINFO_KEY = "hinfo_key";
```

Comment:
> I also wanted to make that parameter private.

which parameter? and why?

> https://github.com/ceph/ceph/blob/master/src/msg/msg_types.h#L40-L46

they are static member variables of `entity_name_t`. and which namespace?